client/my-sites/post-selector/selector.jsx

```diff
@@ -44,21 +49,21 @@ const PostSelectorPosts = React.createClass( {
 		posts: PropTypes.array,
 		postsHierarchy: PropTypes.array,
 		page: PropTypes.number,
-		lastPage: PropTypes.bool,
+		lastPage: PropTypes.number,
```

Comment:
@dmsnell : At this point, I might admit there can be confusion around naming of props without the `is` prefix 😄  (previously indicating equivalent of `isLastPage`, now the actual page number).